app/views/report/_report_modal.erb

```diff
@@ -0,0 +1,23 @@
+<div class="modal fade" id="reportModal" tabindex="-1" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button class="close" data-dismiss="modal" type="button">×</button>
+        <h3 class="modal-title"><%= t("report.modal.title", pod_name: pod_name) %></h3>
+      </div>
+      <div class="modal-body">
+        <% if AppConfig.settings.terms.enable? %>
+          <p><%= t("report.modal.report_only_if_violated_terms") %> <%= link_to(t("report.modal.terms_link", pod_name: pod_name), terms_path, target: "_blank") %></a>.</p>
+        <% end %>
+        <p><%= t("report.modal.you_can_block") %></p>
+        <form id="report-content-form" accept-charset="UTF-8">
+          <p><label for="report-reason-field"><%= t("report.modal.reason_label") %></label></p>
+          <p><textarea id="report-reason-field" class="form-control" placeholder="<%= t('report.modal.placeholder') %>" autofocus required></textarea></p>
```

Comment:
I don't think it really is important actually.

Reply:
But if it's not working, it can also be removed again?

And it might not be super important, but it's still nice to have, and I think it shouldn't be hard to set focus with javascript when opening the modal?